### DISCUSSION TITLE
Security configuration

### DISCUSSION THREAD

**Discussion by mlayou** [android/nowinandroid#1072] [Q&A]
Where is security configured in the Now in Android project?

Thanks.

**Comment by SimonMarquis** [comment]
@mlayou Can you elaborate?
This application currently does not use/store private data, so I don't think there is a need for specific "security" on top of what the Android platform already provides (like [Cleartext / Plaintext traffic](https://developer.android.com/privacy-and-security/risks/cleartext))

**Reply by mlayou** [reply]
I was actually referring to parameters related to secure ranging, like the STS mode, the ranging session key, the ranging preamble, etc.

**Reply by SimonMarquis** [reply]
Unfortunately, this has nothing to do with this project 🙈
